hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenAirdropValidator.java

```diff
@@ -97,11 +98,9 @@ public void validateSemantics(
             @NonNull final ReadableTokenStore tokenStore,
             @NonNull final ReadableTokenRelationStore tokenRelStore,
             @NonNull final ReadableNftStore nftStore) {
-        var tokensConfig = context.configuration().getConfigData(TokensConfig.class);
-        validateTrue(
-                op.tokenTransfers().size() <= tokensConfig.maxAllowedAirdropTransfersPerTx(),
-                TOKEN_REFERENCE_LIST_SIZE_LIMIT_EXCEEDED);
-
```

Comment:
Isn't this check still needed? The check below is using `ledgerConfig.tokenTransfersMaxLen()` and `nftTransfersMaxLen()`, which are both different than `maxAllowedAirdropTransfersPerTx`